src/julia_internal.h

```diff
@@ -183,8 +188,8 @@ extern volatile int jl_in_stackwalk;
 typedef unw_context_t *bt_context_t;
 #endif
 #define MAX_BT_SIZE 80000
-extern ptrint_t bt_data[MAX_BT_SIZE+1];
-extern size_t bt_size;
+extern JL_THREAD ptrint_t bt_data[MAX_BT_SIZE+1];
+extern JL_THREAD size_t bt_size;
```

Comment:
I removed `JL_THREAD` in #14083 since it's not used anymore. Should I add it back or should these go into the tls struct as well?